.github/workflows/integrate.yaml

```diff
@@ -27,7 +27,7 @@ jobs:
 
     steps:
     - name: Check out code
-      uses: actions/checkout@v3
+      uses: actions/checkout@v4
```

Comment:
Ideally these changes would've been added in a separate PR. Not a big deal for this one, but let's keep it in mind for future PRs.

Reply:
yes, I agree and I also wrote that in MM. But to speed up the process here I bundled in the same PR, but we can still use the "rebase & merge" or "merge commit" to preserve the two atomic changes.